Formula/bareos-fd.rb

```diff
@@ -0,0 +1,72 @@
+class BareosFd < Formula
+  desc "Open-Source Data Protection Solution"
+  homepage "https://bareos.org"
+  url "https://github.com/bareos/bareos/archive/Release/17.2.4.tar.gz"
+  sha256 "4c443539012cf5ebb0fdb18878e604e82b951e6429c618acd18762f3c5724799"
+
+  depends_on "readline"
+  depends_on "openssl"
+
+  def install
+    system "./configure",
+    "--prefix=#{prefix}",
+    "--sbindir=#{bin}",
+    "--with-working-dir=#{var}/lib/bareos",
+    "--with-archivedir=#{var}/bareos",
+    "--with-confdir=#{etc}/bareos",
+    "--with-configtemplatedir=#{lib}/bareos/defaultconfigs",
+    "--with-scriptdir=#{lib}/bareos/scripts",
```

Comment:
ditto

Reply:
Would also use `-with-confdir` path.  Scripts are dependent of bareos version and therefore should be located inside of cellar.

Reply:
ok